### PR TITLE
let examples use location obj

### PR DESCRIPTION
**Pull request by derhuerst**
follow-up for #25

### CODE REVIEW

**Comment by derhuerst** [comment]
@kiliankoe want to have a quick look? Julius seems busy these days.

File: examples/invalid-journey.json [hunk 1]
```diff
@@ -15,10 +16,15 @@
 			"type": "station",
 			"id": null,
 			"name": "Baz Main Station",
-			"coordinates": {
+			"location": {
+				"type": "loc",
```
Comment (kiliankoe) [inline examples/invalid-journey.json]: is `loc` valid as well? Or should this be `location` instead?

Comment (derhuerst) [reply examples/invalid-journey.json]: After looking at my own PR again, I had also quickly stumbled over it. It's the invalid example though, so it's supposed not to be `location`. 😉

Comment (kiliankoe) [reply examples/invalid-journey.json]: Oh 🙈

**Comment by kiliankoe** [comment]
LGTM 😊

**Pull request merged**